### DISCUSSION TITLE
Update license holders to be more inclusive of the community and contributors.

### DISCUSSION THREAD

**Discussion by ttytm** [vlang/v#21723] [General]
Some References:

- Go: `[...] The Go Authors [...]` https://github.com/golang/go/blob/e8ee1dc4f9e2632ba1018610d1a1187743ae397f/LICENSE#L1
- Rust: `[...] Copyrights in the Rust project are retained by their contributors. No
copyright assignment is required to contribute to the Rust project.[...]` https://github.com/rust-lang/rust/blob/06c072f158c91296a2f802f2a10c9fe2499f314d/COPYRIGHT#L9-L10
- Kotlin: `[...] and Kotlin Programming Language contributors [...]` https://github.com/JetBrains/kotlin/blob/8f47ea5f6a805d0a871ca4fb51b5b26e96f549d0/license/COPYRIGHT.txt#L2
- Swift: Uses Apache(omits copyright hodlers by default and specifies contributors as licensor):
  `[...] "Licensor" shall mean the copyright owner [...] "Contributor" shall mean Licensor [...]` https://github.com/swiftlang/swift/blob/179a1253a76c1a0e179ab78344abcd332c50c710/LICENSE.txt#L62
- Zig: `[...] Zig contributors [...]` https://github.com/ziglang/zig/blob/ab4c461b76ff7b1d10e6d2010370ea0984f97efe/LICENSE#L3

License headers in code files that are not based on other codebases (such as HTTP parts based on Go) could be omitted to reflect that (they are usually not required and the project's license applies).

**Comment by Wajinn** [comment]
Something that I see as a possible issue, is the nature of Go, Rust, Kotlin... These were languages initially created for the purposes of a specific large corporation and their licenses are not MIT or not solely using it. 

The corporation or foundation is exerting control over the language, which is in parallel or can be of divergent purposes to that of the designers, contributors, and/or the wider community. In other words, some of the examples given, may debatably not be appropriate for V or could cause unforeseen or surprise issues later for the wider community. Probably want to be very careful.

Seems like this ([10025](https://github.com/vlang/v/discussions/10025)), should be linked too.
